app/src/pages/home/index.tsx

```diff
@@ -32,16 +31,26 @@ const Header = () => {
             <Text fontSize={"2xl"}>Manage, Learn, Develop</Text>
           </Box>
           <Stack direction={"row"} spacing={7}>
-            <Link to={"/home"}>
-              <Button colorScheme={"teal"} size={"md"}>
+            <NextLink href={"/train"}>
+              <ChakraLink
+                to={"/train"}
+                backgroundColor="teal"
+                as={Button}
+                size={"md"}
+              >
                 Attendance & Learn
-              </Button>
-            </Link>
-            <Link to={"/home"}>
-              <Button colorScheme={"green"} size={"md"}>
+              </ChakraLink>
+            </NextLink>
+            <NextLink href={"/home"}>
+              <ChakraLink
+                to={"/account/signup"}
```

Comment:
the inner ChakraLink dont need to="" and the links don't even match

see https://nextjs.org/docs/api-reference/next/link